src/panels/config/core/ha-config-name-form.ts

```diff
@@ -40,7 +40,7 @@ class ConfigNameForm extends LitElement {
             )}
             .disabled=${disabled}
             .value=${this._nameValue}
-            @changed=${this._handleChange}
+            @input=${this._handleChange}
```

Comment:
```suggestion
            @change=${this._handleChange}
```
I dont think we need @input but either works